### PR TITLE
Complete skeleton for all morphological categories

### PR DESCRIPTION
**Pull request by HactarCE**
Please proofread these if you can.

### CODE REVIEW

File: yaml/morphology/register.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+name: register
+# TODO Is this an accurate brief for Register?
+brief: relation of utterance to context
```
Comment (porpoiseless) [inline yaml/morphology/register.yaml:4]: It's OK but I kind of dislike using the names of categories in `brief` or `full` unless the names _actually_ refer to those categories. (Like the way EVERY Function in Ithkuil _except_ Manifestive includes the word, "manifest"!) I can't think of anything better at the moment though.

File: README.org [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #+title: Libre TNIL documentation
 * Overview
 [[http://ithkuil.net/][Ithkuil]] is a constructed language created by John Quijada, and TNIL
-("The Next Ithkuilic Language") is a successor language sharing many
+("The New Ithkuilic Language") is a successor language sharing many
```
Comment (porpoiseless) [inline README.org]: Haha, right you are!

File: yaml/morphology/register.yaml (hunk 1 above)
Comment (HactarCE) [reply yaml/morphology/register.yaml:4]: This is why I think we should make a point of capitalizing category and value names. I'll make an issue for that now.

**Comment by HactarCE** [comment]
Ok well I didn't realize all my new commits would be added to this PR. Sorry about that!

**Review by porpoiseless — APPROVED** [review]
Everything looks good!

**Pull request merged**